types/Types.d.ts

```diff
@@ -0,0 +1,369 @@
+interface CommonPlaidLinkOptions {
+    onSuccess: Function;
+    onExit?: Function;
+    logLevel?: LinkLogLevel;
+    extraParams?: Record<string, any>;
+}
+
+export type LinkTokenConfiguration = (CommonPlaidLinkOptions & {
+    token: string;
+});
+
+export type LinkPublicKeyConfiguration = (CommonPlaidLinkOptions & {
+    publicKey: string;
+    token?: string;
+    clientName: string;
+    environment: PlaidEnvironment;
+    product: Array<PlaidProduct>;
+    countryCodes?: Array<string>;
```

Comment:
Should be non null (is non null on both Android and iOS
```suggestion
    countryCodes: Array<string>;
```